release/cloud-build-configs/kfserving-controller.cloud-build.yaml

```diff
@@ -0,0 +1,8 @@
+steps:
+- name: 'gcr.io/cloud-builders/go'
+  args: ['install', './cmd/manager']
+  env: 
+    - 'PROJECT_ROOT=github.com/kubeflow/kfserving'
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '-t', 'gcr.io/kfserving/kfserving-controller-${_VERSION}', '.']
+images: ['gcr.io/kfserving/kfserving-controller-${_VERSION}']
```

Comment:
Where is _VERSION defined ?

Reply:
Passed in through cloud build. I think I'll actually use our git branch for this which will help with versioning in the future.